tests/utils/test_db.py

```diff
@@ -251,3 +258,10 @@ def test_alembic_configuration(self):
         import airflow
 
         assert config.config_file_name == os.path.join(os.path.dirname(airflow.__file__), "alembic.ini")
+
+    @mock.patch("sqlalchemy.orm.Session.scalar")
+    def test_bool_lazy_select_sequence(self, mock_scalar):
+        mock_scalar.return_value = None
+
+        lss = EmptyLazySelectSequence()
+        assert not bool(lss)
```

Comment:
This doesn’t test anything… I believe it passes even without the change.

Reply:
You're right. I've fixed the test and it is now meaningful. Without the `bool()` cast in `check_query_exists` the test fails